packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js

```diff
@@ -666,21 +669,30 @@ export function buildRootObject(vatPowers, vatParameters) {
           // Allow some hardcoded client address connections into the chain.
           // This is necessary for fake-chain, which does not have Cosmos SDK
           // transactions to provision its client.
+          let nonce = 0;
           const demoProvider = Far('demoProvider', {
             // build a chain-side bundle for a client.
             async getDemoBundle(nickname) {
+              nonce += 1;
```

Comment:
This is more than a refactor, right? It seems to have observable differences in behavior.

Reply:
I'll update this to a "fix".